SES1B/Models/Reservation.cs

```diff
@@ -0,0 +1,14 @@
+using System;
```

Comment:
Make sure you also have code in the model builder which is in ResturantDB.context - reference the Model Builder for MenuItems to see how its done. Add a reservation in the database, then see if you can test if the API is running by visiting the route (http://localhost:44349/api/reservation), if you see your entry you know its working